descriptor.md

```diff
@@ -55,13 +55,10 @@ The following fields contain the primary properties that constitute a Descriptor
 - **`artifactType`** *string*
 
   This OPTIONAL property contains the type of an artifact when the descriptor points to an artifact.
-  This is the value of `artifactType` when the descriptor references an [artifact manifest](artifact.md).
   This is the value of the config descriptor `mediaType` when the descriptor references an [image manifest](manifest.md).
 
 Descriptors pointing to [`application/vnd.oci.image.manifest.v1+json`](manifest.md) SHOULD include the extended field `platform`, see [Image Index Property Descriptions](image-index.md#image-index-property-descriptions) for details.
 
-Descriptors pointing to [`application/vnd.oci.artifact.manifest.v1+json`](artifact.md) SHOULD include the extended field `artifactType`.
```

Comment:
Regression versus Proposal E, should this be projected from the config mediatype, matching the referrers API behavior in distribution-spec 1.1-rc.1?

> The descriptors MUST include an `artifactType` field that is set to the value of `artifactType` for an artifact manifest if present, or the configuration descriptor's `mediaType` for an image manifest.

Reply:
IMO, I would love to see this language modified to `MUST`. That aligns with the distribution spec, descriptors are otherwise inconsistent.

Reply:
Policies/github-anti-bribery-statement.md